scanner/go.mod

```diff
@@ -3,20 +3,27 @@ module github.com/stackrox/stackrox/scanner/v4
 go 1.20
 
 require (
+	github.com/google/go-containerregistry v0.15.2
 	github.com/prometheus/client_golang v1.15.1
 	github.com/quay/claircore v1.5.7
 	github.com/quay/zlog v1.1.4
-	github.com/stackrox/rox v0.0.0
```

Comment:
when using `go mod tidy`, this is removed because it's unused. Let's keep this out, but we can re-add it once it's needed